shortcircuit-api/src/main/java/com/powsybl/shortcircuit/json/FaultResultSerializer.java

```diff
@@ -20,6 +20,10 @@
  */
 public class FaultResultSerializer extends StdSerializer<FaultResult> {
 
+    // VERSION = 1.0 fault, shortCircuitPower, timeConstant, feederResult, limitViolations, current, voltage, shortCircuitBusResults
+    // VERSION = 1.1 status
+    private static final String VERSION = "1.1";
```

Comment:
I think it's better to not introduce a version at this level of detail:
let's keep a version at the global shortcircuit result level only (and increment it from 1.0 to to 1.1)

Reply:
Note: before actually doing this change, I would like some confirmation / agreement from @miovd and @geofjamg .

I am not 100% sure there is a consensus about this approach of versioning at the "file" level instead of the "object" level.

Reply:
I agree